get-cloudify.py

```diff
@@ -225,11 +232,11 @@ def install_module(module, version=False, pre=False, virtualenv_path=False,
     if upgrade:
         pip_cmd.append('--upgrade')
     if IS_VIRTUALENV and not virtualenv_path:
-        lgr.info('Installing within current virtualenv: {0}...'.format(
+        logger.info('Installing within current virtualenv: {0}...'.format(
             IS_VIRTUALENV))
     result = run(' '.join(pip_cmd))
     if not result.returncode == 0:
-        lgr.error(result.aggr_stdout)
+        logger.error(result.aggr_stdout)
         sys.exit('Could not install module: {0}.'.format(module))
```

Comment:
for easier testing and management of error codes, I suggest we use this something like:
https://github.com/cloudify-cosmo/wagon/blob/master/wagon/codes.py
but in-script.